scripts/spack/packages/spheral/package.py

```diff
@@ -34,13 +34,13 @@ class Spheral(CachedCMakePackage, CudaPackage):
     # DEPENDS
     # -------------------------------------------------------------------------
     depends_on('mpi', when='+mpi')
-    depends_on('cmake@3.10.0:', type='build')
-
-    depends_on('zlib@1.2.11 +shared +pic', type='build')
+    depends_on('cmake@3.18.0:', type='build')
 
     depends_on('boost@1.74.0 +system +filesystem -atomic -container -coroutine -chrono -context -date_time -exception -fiber -graph -iostreams -locale -log -math -mpi -program_options -python -random -regex -test -thread -timer -wave +pic', type='build')
```

Comment:
Will the Boost version be updated in this branch?

Reply:
No. Is that something you need updated?

Reply:
Were hoping to do a separate PR soon where we can update TPLs to more recent versions. This PR only updates TPLs in order to make the latest version of spack happy.